develop/dev-guide-read-in-fast-mode.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Fast Mode
+summary: Introduces a way to speed up the querying of AP scenarios by using Fast Mode.
+---
+
+# Fast Mode
+
+This document describes how to use Fast Mode to speed up queries in Analytical Processing (AP) scenarios.
+
+TiFlash supports the following modes:
+
+- `Normal Mode`. The default mode. This mode guarantees the accuracy of query results and data consistency.
+- `Fast Mode`. This mode does not guarantee the accuracy of query results and data consistency, but provides more efficient query performance.
+
+Some AP scenarios allow for some tolerance to the accuracy of the query results. In these cases, if you need higher query performance, you can switch the corresponding table to TiFlash's Fast Mode for querying.
+
+The mode switch takes effect globally. TiFlash mode switching is only available for tables with TiFlash Replica. TiFlash-related operations are not supported for temporary tables, in-memory tables, system tables, and tables with non-utf-8 characters in the column names, and therefore changing TiFlash table mode is not supported for them.
```

Comment:
The mode switch  takes effect globally only for the table whose mode are set to fast by the sql ’alter table set tiflash mode‘, not all the tiflash tables. 
Make sure users will not be confused.

Reply:
OK. Refined as follows. PTAL.